modules/lang-painless/src/main/java/org/elasticsearch/painless/DefBootstrap.java

```diff
@@ -2,6 +2,14 @@
 
 import org.elasticsearch.common.SuppressForbidden;
 
+import java.lang.invoke.CallSite;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodHandles.Lookup;
+import java.lang.invoke.MethodType;
+import java.lang.invoke.MutableCallSite;
+import java.lang.invoke.WrongMethodTypeException;
+
```

Comment:
The license header seems to be messed up in this file. It should be before the package statement.

Reply:
Huh, this file hasn't been modified much so it's probably legacy.  Will fix.

Reply:
Fixed.